src/System.ValueTuple/src/System/ValueTuple/ValueTuple.cs

```diff
@@ -405,7 +411,12 @@ int ITupleInternal.GetHashCode(IEqualityComparer comparer)
         /// </remarks>
         public override string ToString()
         {
-            return "(" + Item1 + ")";
+            return "(" + Item1.ToString() + ")";
```

Comment:
String interpolation has quite different (and more complicated) semantics compared to string concatenation. See https://github.com/dotnet/roslyn/issues/9212#issuecomment-189170257